src/metaschema/oscal_component_metaschema.xml

```diff
@@ -232,8 +232,6 @@
             &allowed-values-component_component_service;
       </allowed-values>
 
-      <expect target="." test="not(exists((.)[not(@type='service')]/protocol))"/>
```

Comment:
Blocking: if we are using this integration harness to track issues with upstream, how are we going to properly manage references to these model changes and how we coordinate upstream with/for them? This is just the first example I see, separate of type alignments in previous diffs in other files, so I think we need to be mindful of how we plan to use this.